### PR TITLE
Added tests for MSFT_xComputer resource

### PR DESCRIPTION
**Pull request by SimonWahlin**
Added tests for MSFT_xComputer resource, all tests are passing.

### CODE REVIEW

File: Tests/xComputermanagement.Tests.ps1 [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+    Describe 'MSFT_xComputer' {
+        
+        $SecPassword = ConvertTo-SecureString -String 'password' -AsPlainText -Force
+        $Credential = New-Object -TypeName System.Management.Automation.PSCredential -ArgumentList 'USER',$SecPassword
+        $NotComputerName  = if($env:COMPUTERNAME -ne 'othername'){'othername'}else{'name'}
+    
+        Context Test-TargetResource {
+            Mock Get-WMIObject {[PSCustomObject]@{DomainName = 'ContosoLtd'}} -ParameterFilter {$Class -eq 'Win32_NTDomain'}
+            It 'Throws if both DomainName and WorkGroupName are specified' {
+                {Test-TargetResource -Name $Env:ComputerName -DomainName 'contoso.com' -WorkGroupName 'workgroup'} | Should Throw
```
Comment (TravisEz13) [inline Tests/xComputermanagement.Tests.ps1]: Suggest verifying the message in the future, not blocking the PR.

File: Tests/xComputermanagement.Tests.ps1 [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+            It 'Changes only ComputerName in Workgroup' {
+                Mock GetComputerDomain {''}
+                Mock Get-WMIObject {[PSCustomObject]@{Domain = 'Contoso';Workgroup='Contoso';PartOfDomain=$false}}
+                Set-TargetResource -Name $NotComputerName | Should BeNullOrEmpty
+                Assert-MockCalled -CommandName Rename-Computer -Exactly 1 -Scope It
+                Assert-MockCalled -CommandName Add-Computer -Exactly 0 -Scope It
+            }
+        }
+    }
+}
```
Comment (TravisEz13) [inline Tests/xComputermanagement.Tests.ps1]: end file with new line per item 5 of the [Style Guidelines] (https://github.com/PowerShell/DscResources/blob/master/StyleGuidelines.md) refered to in the contribution guidelines.

**Comment by SimonWahlin** [comment]
I see, sorry for any inconvenience.

**Comment by TravisEz13** [comment]
This is great, thanks for the contribution.  Tests really help maintain quality.

**Pull request merged**